packages/commonwealth/server/routing/router.ts

```diff
@@ -800,6 +804,14 @@ function setupRouter(
     threadsUsersCountAndAvatars.bind(this, models),
   );
 
+  // tags
+  registerRoute(
```

Comment:
Not a big deal, but we are trying to deprecate the old express routes and move towards trpc routes. You can check server/api/index.ts for an example of how we do them. Although this is not really required and since you already wrote them in the express way it is probably fine to keep it as is.

Reply:
Appreciate the insights. Would love to do that, but a little short on delivering this feature so will bump for now.